src/main/java/com/tpark/tournament/entity/Sport.java

```diff
@@ -37,6 +38,6 @@ public long getSportId() {
 
     @Override
     public String toString() {
-        return String.format("Sport[id=%d name=%s]", sportId, name);
+        return "Sport{" + "sportId=" + sportId + ", name='" + name + '\'' + '}';
```

Comment:
Couldnt remember exactly why. But my guess is something to do with %d and %s. I could change it back it if you feel strongly. Leaving it as is for now.